Hazel/src/Hazel/Log.h

```diff
@@ -20,16 +20,22 @@ namespace Hazel {
 
 }
 
-// Core log macros
-#define HZ_CORE_TRACE(...)    ::Hazel::Log::GetCoreLogger()->trace(__VA_ARGS__)
-#define HZ_CORE_INFO(...)     ::Hazel::Log::GetCoreLogger()->info(__VA_ARGS__)
-#define HZ_CORE_WARN(...)     ::Hazel::Log::GetCoreLogger()->warn(__VA_ARGS__)
-#define HZ_CORE_ERROR(...)    ::Hazel::Log::GetCoreLogger()->error(__VA_ARGS__)
-#define HZ_CORE_CRITICAL(...) ::Hazel::Log::GetCoreLogger()->critical(__VA_ARGS__)
-
-// Client log macros
-#define HZ_TRACE(...)         ::Hazel::Log::GetClientLogger()->trace(__VA_ARGS__)
-#define HZ_INFO(...)          ::Hazel::Log::GetClientLogger()->info(__VA_ARGS__)
-#define HZ_WARN(...)          ::Hazel::Log::GetClientLogger()->warn(__VA_ARGS__)
-#define HZ_ERROR(...)         ::Hazel::Log::GetClientLogger()->error(__VA_ARGS__)
-#define HZ_CRITICAL(...)      ::Hazel::Log::GetClientLogger()->critical(__VA_ARGS__)
+#ifdef HZ_BUILD_DLL
```

Comment:
Not sure if we should be using this define still, seeing as we are statically linking now and we aren't actually building a dll. Maybe we can introduce a new define (in premake5.lua) for this? Something like just `HZ_BUILD`.

Reply:
It still worked as intended for me. However if it makes more sense to change it then I agree.

Reply:
I would suggest `HZ_BUILD_ENGINE` and derive the (unused) DLL part from that.

Reply:
I would suggest HZ_BUILD_ENGINE and derive the (unused) DLL part from that.

Reply:
I would suggest HZ_BUILD_ENGINE and derive the (unused) DLL part from that.

Reply:
Agreeing with this.

Reply:
I’ll make this change next time I have a chance.